HMCL/src/main/resources/assets/lang/I18N.properties

```
@@ -1144,6 +1144,30 @@ repositories.chooser.title=Choose download source for JavaFX
 resourcepack=Resource Packs
 resourcepack.download.title=Download Resource Pack - %1s
 
+reveal.in_explorer=Reveal in explorer
+
+schematics=Schematics
+schematics.add=Add schematic files
+schematics.add.failed=Failed to add schematic files
+schematics.back_to=Back to "%s"
+schematics.create_directory=Create Folder
+schematics.create_directory.prompt=Please enter the new folder name
+schematics.create_directory.failed=Failed to create folder
+schematics.create_directory.failed.already_exists=Folder already exists
+schematics.create_directory.failed.empty_name=Name cannot be empty
+schematics.create_directory.failed.invalid_name=Name contains invalid characters
+schematics.info.description=Description
+schematics.info.enclosing_size=Enclosing Size
+schematics.info.name=Name
+schematics.info.region_count=Regions
+schematics.info.schematic_author=Author
+schematics.info.time_created=Created Time
+schematics.info.time_modified=Modified Time
+schematics.info.total_blocks=Total Blocks
+schematics.info.total_volume=Total Volume
+schematics.manage=Schematics Management
```

*It's too long and will be partially obscured.*

```suggestion
schematics.manage=Schematics
```